Countly.js

```diff
@@ -24,8 +24,14 @@ if (Platform.OS.match("android")) {
 }
 
 // countly initialization
-Countly.init = function(serverUrl, appKey, deviceId = ""){
+Countly.init = async function(serverUrl, appKey, deviceId = ""){
 
+    if(deviceId == "") {
+        deviceId = null;
+        if(await CountlyReactNative.isLoggingEnabled()) {
+            console.error("[CountlyReactNative] init, Device Id is empty");
```

Comment:
set the message after the comma to "Device Id during init can't be empty string"